lib/OcrProcessors/PdfOcrProcessor.php

```diff
@@ -40,14 +45,20 @@ public function ocrFile(string $fileContent): string {
 			->setStdIn($fileContent);
 
 		$success = $this->command->execute();
+
 		$errorOutput = $this->command->getError();
 		$stdErr = $this->command->getStdErr();
 		$exitCode = $this->command->getExitCode();
 
-		if ($success && $errorOutput === '' && $stdErr === '') {
+		if ($success) {
+			if ($stdErr !== '' || $errorOutput != '') {
```

Comment:
Why are you using different types of comparison operators?

According to the [API documentation from php-shellcommand](https://github.com/mikehaertl/php-shellcommand#methods), both return values contain either the error message or are empty if no error occurred. So we should handle both return values equally.

Reply:
Wasn't my intention, changed to strict comparison (see https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison) and force pushed.